spec/support/command_helpers.rb

```diff
@@ -64,7 +69,7 @@ def dummy_test_app_prefix(extra_prefix = "")
   # Returns the app name.
   def dummy_test_app(extra_prefix = "", suffix = "", create_if_not_exists: false) # rubocop:disable Metrics/CyclomaticComplexity
     prefix = dummy_test_app_prefix(extra_prefix)
-    suffix = SecureRandom.hex(4) if (suffix.nil? || suffix.empty?) && !create_if_not_exists
+    suffix = SecureRandom.hex(2) if (suffix.nil? || suffix.empty?) && !create_if_not_exists
```

Comment:
Reduced the size so that the names are shorter, because Control Plane has a limit of 64 characters.

It should still be rare to run into conflicts, so I don't think we need to worry about that for now.